test/no-function-overriding.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * @fileoverview Test for no-function-overriding rule
+ * @author Nicolas Feignon <nfeignon@gmail.com>
+ */
+
+'use strict';
+
+var Solium = require ('solium');
+
+var userConfig = {
+    rules: {
+        "security/no-function-overriding": "error"
+    }
+};
+
+describe ('[RULE] no-function-overriding: Acceptances', function () {
+
+    it ('should accept contracts that don\'t override their inherited functions', function (done) {
```

Comment:
Are modifiers part of a function's signature? ie, is below example considered function overriding:
```
contract A {
  function a () payable {}
  function b () myOwnModifer {}
  function c () returns (bytes32, uint) {}
}

contract B is A {
  function a() {}
  function b() {}
  function c() {}
}
```

Reply:
Modifiers aren't part of a function's signature, only its parameters.
From the [solidity documentation](http://solidity.readthedocs.io):
>    // Functions can be overridden by another function with the same name and
     // the same number/types of inputs.  If the overriding function has different
     // types of output parameters, that causes an error.
     // Both local and message-based function calls take these overrides
     // into account.

I tried in Remix, it causes an compilation error if I try to override a function but change its modifiers.

I'll add some tests to account for this though.